src/base/convert.coffee

```diff
@@ -10,8 +10,12 @@ exports.extractValue = (callback) ->
 			else
 				messageToUse = messages
 			debug "message to use",messageToUse
-			result = messageToUse[0].payload.replace(new RegExp(" ", "g"), "")
-			callback(error, result)
+			if messages.length > 1
```

Comment:
Hi!
Thanks for the pull request. Please keep an eye on the failing travis ci build.
This should be >=1, otherwise results with exaclty one message will result in an error. This is the reason why the travis ci build for this pull request fails.
I will merge the PR when the tests are green.

Regards, Ben